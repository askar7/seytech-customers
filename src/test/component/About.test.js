import React from 'react';
import About from '../../components/About';
import { mount, shallow } from 'enzyme';
import renderer from 'react-test-renderer'; // snapshot testing

describe('App tests', () => {
  // snapshot
  test('renders snapshot', () => {
    const app = renderer.create(<About />);
    const tree = app.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders first header element correctly', () => {
    let app = mount(<About />);
    expect(app.find('h1').text()).toBe('Seytech');
  });
});
