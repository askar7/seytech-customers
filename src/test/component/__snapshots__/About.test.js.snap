// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests renders snapshot 1`] = `
<div>
  <h1>
    Seytech
  </h1>
  <p>
    Fullstack Software Developer Bootcamp
  </p>
</div>
`;
